hubble/utils/auth.py

```diff
@@ -21,60 +22,37 @@ def get_auth_token():
         return token_from_env if token_from_env else config.get('auth_token')
 
     @staticmethod
-    async def login():
+    async def login(**kwargs):
         api_host = get_base_url()
-
+        auth_info = None
         async with aiohttp.ClientSession(trust_env=True) as session:
-            redirect_url = 'http://localhost:8085'
+            kwargs['redirect_url'] = 'https://hub.jina.ai/authorization-landing'
+            kwargs['provider'] = 'jina-login'
```

Comment:
The idea is to allow the application to specify custom `provider` and `app`
```suggestion
            kwargs['provider'] = kwargs['provider'] or 'jina-login'
```